app/lib/staypuft/seeder.rb

```diff
@@ -342,7 +342,7 @@ def functional_dependencies
       ceph_cluster_network      = { :string => "<%= @host.network_query.network_address_for_host('#{Staypuft::SubnetType::STORAGE_CLUSTERING}') %>" }
       # FIXME: this should actually be STORAGE instead of PXE, but only after we have a reliable way of identifying DNS names
       #        on the storage network
```

Comment:
If you don't mind, can you delete this FIXME comment while you're at it -- since we're making that change now?